packages/calcite-components/support/syncEnT9nBundles.ts

```diff
@@ -13,18 +11,19 @@
 
   for (const entry of entries) {
     const path = entry.split(win32.sep).join(sep);
-    const component = path.split(sep)[2];
+    const componentName = path.split(sep).at(-3);
+    const monorepoRoot = join(import.meta.dirname, "../../..");
 
-    const source = resolve(`${path}/messages.json`);
-    const destination = resolve(`${path}/messages.en.json`);
+    const source = resolve(monorepoRoot, `${path}/messages.json`);
+    const destination = resolve(monorepoRoot, `${path}/messages.en.json`);
 
     await copyFile(source, destination);
-    synchronized.push(component);
+    synchronized.push(componentName);
```

Comment:
This is causing a type error. .at( ) return value can be ` string| undefined`.

Reply:
Thanks for the catch! 🎣 The error wasn't showing up because I hadn't synced up w/ `dev`.